packages/fluentui/react-builder/src/components/MessageComputer.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ TODO:
+* Add the missing and not so obvious attributes (e.g. "aria-haspopup" or "aria-expanded") for the already defined roles (e.g. "menuitem" or "checkbox") according to the specification.
+* With JAWS, in the case of the element with the "listbox" role, differentiate between having and not having the aria-multiselectable="true" attribute. If this attribute is present, then aria-selected="false" on the child elements with the role "option" behave differently than if it is not present. Specifically, if aria-multiselectable="true" is present, the aria-selected="false" causes the narration of "not selected", but if present, having the aria-selected attribute makes no difference to the narration. 
+ * What about the "disabled" state?
+ */
+import SRMC from './SRMC-Definitions';
+import './SRMC-Rules-Win_JAWS';
+
+export default class MessageComputer {
+  private computedMessageParts: { [key: string]: string } = {
+    value: '',
+    name: '',
+    description: '',
+    type: '',
+    state: '',
+    position: '',
+    usage: '',
+  };
+
+  // Asynchronously computes and returns the complete screen reader message for the given element and platform.
+  async computeMessage(element: HTMLElement, platform: string): Promise<string> {
+    let definitionName = this.getDefinitionName(element, platform, 'stateRules');
+
+    // Determine and save the usage string
+    this.computedMessageParts.usage = '';
+    const usages = SRMC.usageStrings[platform][definitionName];
+    if (usages) {
+      // Begin if 1
+      if (usages['[default]']) {
+        // Begin if 2
+        this.computedMessageParts.usage = usages['[default]'];
+      } // End if 2
+      for (let usageName in usages) {
+        // Begin for 1
+        const split = usageName.split('=');
+        const state = split[0];
+        const stateValue = split[1];
```

Comment:
```suggestion
        const [state, stateValue] = usageName.split('=')
```

adding @ling1726  suggestion just to make it simpler to apply :)